compiling-source/sass/components/_navbar.scss

```diff
@@ -6,14 +6,44 @@ nav {
   }
 
   // Logo
-  .brand-logo {
-    &:not(.center) {
-      margin-left: 20px;
+  .nav-logo {
+    position: absolute;
+    color: $navbar-font-color;
+    display: inline-block;
+    font-size: $navbar-brand-font-size;
+    padding: 0;
+    margin-left: 20px;
+    &.center {
+      left: 50%;
+      transform: translateX(-50%);
     }
 
-    @media #{$small-only} {
+    @media only screen and (max-width: 783px) {
```

Comment:
As stated before: use the parameters instead of hard-coding these things.
Because currently this even causes a 1xp problem when you have a window of exactly 783px, the logo will be centered but the view didn't switch yet to the mobile view (because that is below 783px).
Now is this obviously not an important problem in this usecase but it causes problems when someone wants to change the screen widths to different sizes and changes the parameter but the logo (not linked to that parameter) is now doing something completely different.
Please be more careful to ALWAYS use the defined variables where possible (and create a variable if usefull).

Reply:
If you want to do that, then you have to override the default materialize variables, since by default materialize does not define any of the sizes which you requested in your previous comment.

Reply:
`@media #{$small-only} {` should work fine, what was wrong with that?

Reply:
Whoops missed that. It has become a bit confusing since you introduced two files which contain a whole bunch of variables. https://github.com/Moridrin/ssv-material/blob/develop/compiling-source/materialize/components/_variables.scss and https://github.com/Moridrin/ssv-material/blob/develop/compiling-source/sass/components/_baseVariables.scss
Perhaps consider using the second one for overrides only.

Reply:
Perhaps, I thought it could be useful to see all variables that are available without having to look at the materialize files (because you want to prevent any editing in those files).

Reply:
The variables in the second file don't match the variables in the first file as they seem to be from two completely different versions of materialize.